### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by Zaxx69** [sayaka-sh/spmp#98] [General]
Can you open a telegram channel to dicuss about this app and make it better, we can share videos about bugs rather than screenshots and users find it easy to talk in telegram than in github and much more, it will be fun. And many ideas will shared

**Comment by sayaka-sh** [comment]
Good idea. I don't use Telegram, but I did made a Discord server instead: https://discord.gg/7RmSgSNVYE
